Elasticity.C

```diff
@@ -1035,12 +1035,6 @@ bool ElasticityNorm::finalizeElement (LocalIntegral& elmInt)
 }
 
 
-void ElasticityNorm::addBoundaryTerms (Vectors& gNorm, double energy) const
```

Comment:
Not any longer. It used to be an abstract interface, but with identical implementations in all the apps using it, so I moved that into the base class instead. Left it as virtual as not 100% cwrtain some apps later might need a different implementation. But that is not likely so we can as well 'devirtualize' it.